server/main.lua

```diff
@@ -39,12 +39,35 @@ if Config.UseWolfknightRadar == true then
 	AddEventHandler("wk:onPlateScanned", function(cam, plate, index)
 		local src = source
 		local Player = QBCore.Functions.GetPlayer(src)
-		local bolo = GetBoloStatus(plate)
+		local PlayerData = GetPlayerData(src)
+		local vehicleowner = GetVehicleOwner(plate)
+		local bolo, title, boloid = GetBoloStatus(plate, title, boloid)

```

Comment:
You don't need to pass in title or boloId arguments, they actually dont even exist at this point so you're just passing two nil values

Reply:
Amended code in new commit d13ea57